### ISSUE TITLE
[FEATURE][MacOS] Open file location & Show file path

### ISSUE THREAD

**Issue by ValentinWY** [strongbox-password-safe/Strongbox#803]
Could be nice to have a button to open the vault file location and a subtitle to show it's path. I'm aware that you can click the the vault name when it's unlocked to view the path, but it could be nice to have it open when you click on the folder (like in the finder app). Additionally you could add a modifier when the command key is held to open the vault file location when you click on it.

**Comment by strongbox-mark** [comment]
Yes, a good idea, a "Show in Finder" menu option at least on the DB Manager. Thanks.
